src/Codeception/Module/Yii2.php

```diff
@@ -628,18 +633,21 @@ public function grabFixture(string $name, null|string $index = null): Fixture|\y
      * $user_id = $I->haveRecord('app\models\User', array('name' => 'Davert'));
      * ?>
      * ```
+     *
      * @template T of \yii\db\ActiveRecord
-     * @param class-string<T> $model
-     * @param array<string, mixed> $attributes
-     * @part orm
+     * @param    class-string<T>      $model
+     * @param    array<string, mixed> $attributes
+     * @part     orm
      */
     public function haveRecord(string $model, $attributes = []): mixed
     {
-        /** @var T $record */
+        /**
+ * @var T $record
+*/
```

Comment:
please tabulate this properly

Reply:
Hmm, that's a CS fixer bug; i'm gonna look into a rule that takes care of it. Thanks.